Move qconfig removal into convert()

hx89 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#41930 Move qconfig removal into convert()**

As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

Commit:
Move qconfig removal into convert()

As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

[ghstack-poisoned]

Commit:
Update on "Move qconfig removal into convert()"


As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

[ghstack-poisoned]

raghuramank100 reviewed (approved):
Is there a test that checks if qconfig is removed after calling convert?

Commit:
Update on "Move qconfig removal into convert()"


As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

[ghstack-poisoned]

Commit:
Update on "Move qconfig removal into convert()"


As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

[ghstack-poisoned]

hx89 commented:
Added extra tests to check if qconfig is removed.

hx89 commented:
> Is there a test that checks if qconfig is removed after calling convert?

Added extra tests to check if qconfig is removed.

Commit:
Update on "Move qconfig removal into convert()"


As title

Differential Revision: [D22698386](https://our.internmc.facebook.com/intern/diff/D22698386/)

[ghstack-poisoned]
